Source/WebCore/css/FontFaceSet.cpp

```diff
@@ -166,6 +167,32 @@ void FontFaceSet::load(const String& font, const String& text, LoadPromise&& pro
     for (auto& face : matchingFaces)
         face.get().load();
 
+    if (scriptExecutionContext() && !scriptExecutionContext()->settingsValues().downloadableBinaryFontsEnabled) {
+        // There are sites (such as HBOMax.com) that expect that loading fonts will succeed,
```

Comment:
Do we know of any sites other than hbo.com? I think if it's just that one site, a quirk would be better. I don't think we should lie about fonts successfully downloading when they have not.